package/Dockerfile

```diff
@@ -40,7 +40,7 @@ ENV KUSTOMIZE_VERSION v3.5.4
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=0.3.500
 ENV CATTLE_RANCHER_OPERATOR_MIN_VERSION=0.1.400
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.000
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.200
```

Comment:
Same question as here: https://github.com/rancher/charts/pull/1458/files#r704828161
Are we confident all of those changes can be backported. We probably should have branched the webhook.

Reply:
The changes include adding the support for hardened cluster and a pre-delete-hook for cleaning up `mutatingwebhookconfigurations`, both of them can be backported. 

We have branched the app,   `0.1.200` is cut from the `release/v0.1` branch for rancher 2.5